src/tools/idlc/src/descriptor.c

```diff
@@ -1287,7 +1287,7 @@ static int print_keys(FILE *fp, struct descriptor *descriptor, bool keylist)
 static int print_flags(FILE *fp, struct descriptor *descriptor)
 {
   const char *fmt;
-  const char *vec[3] = { NULL, NULL, NULL };
+  const char *vec[4] = { NULL, NULL, NULL, NULL };
```

Comment:
I'd do `const char *vec[4] = { NULL };` because that guarantees all remaining entries in `vec` are initialized to a null pointer as well.

Reply:
Yep.